api/api/src/main/scala/uk/ac/wellcome/platform/api/elasticsearch/WorksQuery.scala

```diff
@@ -34,17 +38,25 @@ case object WorksMultiMatcher {
             (Some(1000), "data.imageData.id.otherIdentifiers.value"),
           ).map(f => FieldWithOptionalBoost(f._2, f._1.map(_.toDouble)))
         ),
-        prefixQuery("data.title.keyword", q).boost(1000),
         MultiMatchQuery(
           q,
-          `type` = Some(CROSS_FIELDS),
+          `type` = Some(BEST_FIELDS),
           operator = Some(AND),
           fields = Seq(
-            (Some(1000), "data.contributors.agent.label"),
             (Some(100), "data.title"),
             (Some(100), "data.title.english"),
             (Some(100), "data.title.shingles"),
             (Some(100), "data.alternativeTitles"),
+            (None, "data.lettering"),
+          ).map(f => FieldWithOptionalBoost(f._2, f._1.map(_.toDouble))),
```

Comment:
Maybe use named variables here? Something like:

```suggestion
          ).map( case (weight, fieldName) => FieldWithOptionalBoost(fieldname, weight.map(_.toDouble))),
```